index.js

```diff
@@ -1,3 +1,21 @@
+$( document ).ready(function() {
```

Comment:
I sort of like the clean JS you have written. Could use some restructuring of CSS for elegance, but later.